client/lib/cart-values/cart-items.js

```diff
@@ -286,6 +286,17 @@ function hasOnlyRenewalItems( cart ) {
 }
 
 /**
+ * Determines whether any product in the specified shopping cart is a renewable subscription.
+ * Will return false if the cart is empty.
+ *
+ * @param {Object} cart - cart as `CartValue` object
+ * @returns {boolean} true if all the products in the cart are of the productSlug type
+ */
+function hasRenewableSubscription( cart ) {
+	return cart.products && some( getAll( cart ), function (cartItem){ return cartItem.bill_period > 0 } );
```

Comment:
Minor thin, according to the coding guidelines: there should be spaces around function parentheses. We can also use ES6 arrow function syntax here:
```js
return cart.products && some( getAll( cart ), cartItem => cartItem.bill_period > 0 );
```